Formula/passenger.rb

```diff
@@ -26,6 +27,21 @@ def install
 
     system "rake", "apache2"
     system "rake", "nginx"
+    nginx_addon_dir = `/usr/bin/ruby ./bin/passenger-config about nginx-addon-dir`.strip
+
+    system "tar", "-xf", Dir["#{Formula["nginx"].opt_share}/src/*nginx*.tar.gz"].first
+    Dir.chdir("nginx-#{Formula["nginx"].installed_version}") do
+      _, stderr, = Open3.capture3("nginx", "-V")
+      args = stderr.split("configure arguments:").last.split(" --").reject(&:empty?).map { |s| "--#{s.strip}" }
```

Comment:
This seems pretty hacky/brittle to me. It's looking a lot like it's not intended to be able to build a nginx passenger module like this after the fact as this doesn't guarantee the same args/flags/compiler are used.

CC @FooBarWidget for thoughts on the best approach to take here.

Reply:
We ship the passenger nginx module for recent ubuntu distros in a similar way. The Ubuntu repo provides the nginx and we build a compatible dynamic module and ship that. As long as configure is called the same way, the make step should use the same flags.

Reply:
The "should" worries me a bit here. I don't think this would be widely used enough to warrant being part of nginx itself. I'll make some more inline suggestions that may make this more robust.

> So what happends when nginx would be updated ? Would the module stop working?

Yes. A `test do` will need to be added to verify this and ensure it is rebuild.

Reply:
I haven't checked the size of the module but I doubt it can be massive but might make a large percentage of nginxs wooping 1.4MB . It seems like its practically a two line addition to the nginx formula (except the test) which would be less than adding the source files (which also consumes space possibly beyond the since of the dynamic module).

Reply:
@javian I'm more concerned about the built-time dependencies and that `nginx` will have to depend on `passenger` which feels a bit odd.

Reply:
But I'm perhaps being irrational here and would welcome thoughts from other @Homebrew/maintainers

Reply:
Just out of curiosity can there be multiple `test do` blocks in a formula or should I just extend the existing one?

Reply:
Extend the existing one, thanks @CamJN.

I think the approach I've described in the PR comments of making it as easy as possible to build an nginx module by providing the source and e.g. configure args so that the passenger formula doesn't need to do much would be acceptable 👍